tests/select_for_update/tests.py

```diff
@@ -148,15 +150,30 @@ def test_for_update_sql_model_inheritance_ptr_generated_of(self):
     def test_for_update_sql_model_inheritance_nested_ptr_generated_of(self):
         with transaction.atomic(), CaptureQueriesContext(connection) as ctx:
             list(EUCity.objects.select_related('country').select_for_update(
-                of=('self', 'country__country_ptr',),
+                of=('self', 'country', 'country__country_ptr',),
```

Comment:
This is not related with using nested `_ptr`, so we should add a new test for this bugfix.